pytorch_lightning/plugins/native_amp.py

```diff
@@ -51,15 +51,19 @@ def backward(self, closure_loss, optimizer, opt_idx, *args, **kwargs):
 
         return closure_loss
 
-    def training_step(self, fx, args):
-        with torch.cuda.amp.autocast():
-            output = fx(*args)
-        return output
-
```

Comment:
where did these guys go?

Reply:
I checked coverage and they weren't used, so I removed them.

Reply:
that doesn't sound correct.

Maybe test this manually? native_amp also has the same (training_step). 
I'm not sure coverage is right here